annotations/cut.json

```diff
@@ -1,33 +1,56 @@
 {
-    "command": "cut",
-    "cases":
-    [
-        {
-            "predicate":
-            {
-                "operator": "and",
-                "operands":
-                [
-                    {
-                        "operator": "val_opt_eq",
-                        "operands": ["-d", "--delimiter", "\n"]
-                    },
-                    {
-                        "operator": "exists",
-                        "operands": ["-f", "--fields"]
-                    }
-                ]
-            },
-            "class": "pure",
-            "inputs": ["stdin"],
-            "outputs": ["stdout"],
-			"comments": "Stateless in all cases with exception in case where newline is a delimiter."
-        },
-        {
-            "predicate": "default",
-            "class": "stateless",
-            "inputs": ["stdin"],
-            "outputs": ["stdout"]
-        }
-    ]
+  "command": "cut",
+  "cases": [
+    {
+      "predicate": {
+        "operator": "or",
+        "operands": [
+          {
+            "operator": "val_opt_eq",
+            "operands": [
+              "-d",
+              "\n"
+            ]
+          },
+          {
+            "operator": "exists",
+            "operands": [
+              "-z"
+            ]
+          }
+        ]
+      },
+      "class": "pure",
+      "inputs": [
+        "args[:]"
+      ],
+      "outputs": [
+        "stdout"
+      ]
+    },
+    {
+      "predicate": "default",
+      "class": "stateless",
+      "inputs": [
+        "args[:]"
+      ],
+      "outputs": [
+        "stdout"
+      ]
+    }
+  ],
+  "options": [
+    "stdin-hyphen",
+    "empty-args-stdin"
+  ],
+  "short-long": [
```

Comment:
This is not yet supported by PaSh. We can open an issue or add a test with a long version of one of these flags and use that in a PR to implement that.